x/storage/client/cli/tx.go

```diff
@@ -516,12 +516,13 @@ func CmdDiscontinueObject() *cobra.Command {
 
 func CmdCreateGroup() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "create-group [group-name] [member-list]",
+		Use:   "create-group [group-name] [member-list [extra]",
```

Comment:
What if a want to create a group with extra but without any member?

Reply:
fixed, 'extra' and 'member' can all be optional